app/src/main/res/values/colors.xml

```diff
@@ -69,4 +69,8 @@
   <color name="catappult_pink">#ff057b</color>
   <color name="appc_orange">#fd6363</color>
 
+  <!--Widgets-->
```

Comment:
why didnt you use the attribute in rigth away where you want to use the color?

Reply:
This was a leftover from a previous test. Removed;